frontend/lib/src/task.dart

```diff
@@ -76,14 +78,28 @@ class _Task extends State<Task> {
                       color: Colors.grey),
                 ),
                 right: 2,
-              )
+              ),
             ]),
             Container(
-              child: Align(
-                alignment: Alignment.centerLeft,
-                child: Text(
-                  widget.description,
-                ),
+              child: Column(
+                children: [
+                  Align(
+                    alignment: Alignment.centerLeft,
+                    child: Text(
+                      widget.description,
+                    ),
+                  ),
+                  Align(
+                    alignment: Alignment.centerLeft,
+                    child: Text(
+                      "Points: $points",
+                      style: const TextStyle(
```

Comment:
not specifically for this PR, but for the future, is it possible for us to look into fetching these styles from a common place, such as something like `Styles.RegularText` that contains all these css?